api/PclusterApiHandler.py

```diff
@@ -641,37 +652,41 @@ def set_user_role():
 
 
 def login():
```

Comment:
issue(blocking): We should add a test for this function, as we started doing with https://github.com/aws-samples/pcluster-manager/pull/172

We could go for an easy first test for this function, for instance:

```
given a request with code
  when the auth provider responds successfully
  it should add the accessToken cookie to the response
  it should add the idToken cookie to the response
  it should add the refreshToken cookie to the response
```

Reply:
sure, will do ASAP